modules/sara_r4/cellular_r4.c

```diff
@@ -490,11 +492,58 @@ uint32_t _Cellular_GetSocketId( CellularContext_t * pContext,
         cellularStatus = CELLULAR_BAD_PARAMETER;
     }
 
-    if( ( cellularStatus == CELLULAR_SUCCESS ) &&
-        ( sessionId >= MIN_TCP_SESSION_ID ) && ( sessionId <= MAX_TCP_SESSION_ID ) )
+    if( ( cellularStatus == CELLULAR_SUCCESS ) && ( sessionId <= MAX_TCP_SESSION_ID ) )
     {
         socketIndex = pModuleContext->pSessionMap[ sessionId ];
     }
 
     return socketIndex;
 }
+
+/*-----------------------------------------------------------*/
+
+uint32_t _Cellular_GetSessionId( CellularContext_t * pContext,
```

Comment:
As discussed, this API should follow the [coding convention from embedded c-sdk](https://docs.aws.amazon.com/embedded-csdk/latest/lib-ref/docs/doxygen/output/html/guide_developer_styleguide.html) since it exist in FreeRTOS repo, but it can be put another PR.

Reply:
Update all in another PR.